### DISCUSSION TITLE
Where can I find the fingerprints for the GPG keys?

### DISCUSSION THREAD

**Discussion by rusty-snake** [terrapkg/packages#2787] [Q&A]
I'm looking for a page like https://rpmfusion.org/keys but terra.

**Comment by beedell-roke** [comment]
@rusty-snake, [`discussions/7736#discussioncomment-15107451`](https://github.com/terrapkg/packages/discussions/7736#discussioncomment-15107451) *might* provide some relevant information:

> Terra's `.repo` file includes a link to the GPG key used for the repository.

**Reply by phoenix-rising823** [reply]
I second @rusty-snake . I'd like to be able to verify the GPG fingerprint before installing any packages from Terra.

**Comment by TheHeroBrine422** [comment]
https://github.com/terrapkg/packages/tree/frawhide/anda/terra/gpg-keys

@phoenix-rising823
